.github/workflows/pr-gate.yml

```diff
@@ -0,0 +1,70 @@
+name: pr-gate
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
the PR could not be within the master branch

Reply:
PR to master, is where we run this.